<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="BoardTab">
        <attribute name="title">
         <string>Board</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Other</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Draw coordinates:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="CoordsCheckBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Board</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="3" column="1">
               <widget class="QComboBox" name="PieceSetList"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Lightsquares color:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Darksquares color:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="DSColorButton">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Field selection color:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="SelectionColorButton">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Piece set:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="LSColorButton">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="EngineTab">
        <attribute name="title">
         <string>Engine integration</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LSColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>lightSquaresColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectionColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>selectionColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>saveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>resetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CoordsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>checkBoxToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PieceSetList</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>pieceSetChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DSColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>darkSquaresColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lightSquaresColorClicked()</slot>
  <slot>darkSquaresColorClicked()</slot>
  <slot>selectionColorClicked()</slot>
  <slot>saveClicked()</slot>
  <slot>resetClicked()</slot>
  <slot>checkBoxToggled(bool)</slot>
  <slot>pieceSetChanged(QString)</slot>
 </slots>
</ui>
